bitnami/postgresql/README.md

```diff
@@ -180,6 +180,7 @@ $ kubectl delete pvc -l release=my-release
 | `service.nodePort`                            | Specify the nodePort value for the LoadBalancer and NodePort service types                                                                                | `""`                        |
 | `service.annotations`                         | Annotations for PostgreSQL service                                                                                                                        | `{}`                        |
 | `service.loadBalancerIP`                      | Load balancer IP if service type is `LoadBalancer`                                                                                                        | `""`                        |
+| `service.externalTrafficPolicy`               | Enable client source IP preservation                                                                            | `Cluster`                      |
```

Comment:
If you make it optional and don't set a default value in the values.yaml, you could leave it as `""`.